### PR TITLE
6 feature 6 admin

### PR DESCRIPTION
**Pull request by enfz**
Admin Feature basic implementation done, but needs to be cleaned.
Screen logic done, but needs button implementation and needs to be cleaned

### CODE REVIEW

File: src/main/java/Admin.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Admin class
+//TODO FIX IMPLEMENTATION TO FOLLOW CLEAN ARCHITECTURE
```
Comment (ericshi03) [inline src/main/java/Admin.java:2]: Unfinished TODO - do you plan on adding an admin interface and some builders?

File: src/main/java/Game.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//TEMPORARY FILE WILL BE OVERWRITTEN BY ACTUAL GAME CLASS
```
Comment (ericshi03) [inline src/main/java/Game.java:1]: You don't have to upload this file if it's going to be replaced.

**Review by ericshi03 — DISMISSED** [review]
Overall code looks good. All files follow clean architecture upon viewing.

**Review by Jefferson-liu — APPROVED** [review]
We will probably have to fix this to be clean architecture but the logic looks fine

**Review by ericshi03 — APPROVED** [review]
Code changes look good.

**Review by Andy-Guo-21 — APPROVED** [review]
Looks good to me. We might need to refactor a bit when we implement the use cases to connect the UI with the entities.

**Pull request merged**